src/Core/src/Platform/iOS/CoreAnimationExtensions.cs

```diff
@@ -37,10 +37,6 @@ internal static Matrix4x4 GetViewTransform(this CALayer layer)
 			if (layer == null)
 				return new Matrix4x4();
 
-			var superLayer = layer.SuperLayer;
-			if (layer.Transform.IsIdentity && superLayer == null)
-				return new Matrix4x4();
-
```

Comment:
This does fix the test and I don't see any visual change in the working of the transformations on iOS.

From what I gather on the line below we're already checking if the `SuperLayer` is null and else just take the identity matrix and apply the transform to whichever one comes out of that null coalescing? 

But happy to have a more experts opinion here. I see @drasticactions did something around this area in #5206